Poul12/SimpleToDoApp/src/main/java/pl/_1024kb/stowarzyszenienaukijavy/simpletodo/controller/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,32 @@
+package pl._1024kb.stowarzyszenienaukijavy.simpletodo.controller.filters;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+public class AuthenticationFilter implements Filter
+{
+	@Override
+	public void init(FilterConfig filterConfig) {
+	}
+
+	@Override
+	public void doFilter(ServletRequest request, ServletResponse response,FilterChain chain) throws IOException, ServletException 
+	{
+		HttpServletRequest httpRequest = (HttpServletRequest) request;
+		HttpServletResponse httpResponse = (HttpServletResponse) response;
+		HttpSession session = httpRequest.getSession(false);
+
+		if(session != null && session.getAttribute("username") != null)
```

Comment:
Nawet jeśli masz jednolinijkowe zagnieżdżenie to i tak stosuj klamry. W przyszłości łatwiej będzie dodawać nowy kod. ;)